lib/release/notes/log.rb

```diff
@@ -81,11 +81,29 @@ def find_all_tags_and_log_all
           next unless previous_tag.present? &&
                       system_log(tag_from: previous_tag, tag_to: ta, label: all_labels).present?
 
-          digest_date date: date_humanized(date: System.tag_date(tag: ta))
+          header_content_all(ta)
           copy_single_tag_of_activity(tag_from: previous_tag, tag_to: ta)
         end
       end
 
+      # @api private
+      def header_content
+        if header_title == "date"
+          digest_header date: date_humanized
+        elsif header_title == "tag"
```

Comment:
We should just have an else here since if it's anything other than "tag" or "date" the user will be very confused. And we should try to abstract this some since you use the same conditional in the method below this one.